wiki/Beatmap_Discussion/en.md

```diff
@@ -165,17 +173,22 @@ Your post will be submitted! You can restart at step 3 if you wish to continue m
 
 ### Is Kudosu! available in Beatmap Discussion?
 
-Kudosu! is not yet implemented in Beatmap Discussion. You will not recieve any for submissions you make in the Discussion section. However, it is unknown if current submissions will receive Kudsou! when it is finally implemented. It is also unknown whether or not Kudosu! will be returning either.
+Kudosu! has been implemented! You can give modders Kudosu! by using the thumbs Up icon on discussions that were marked as Suggestions or Problems.
```

Comment:
Kudosu! has been recently implemented in the Beatmap Discussion! You can give modders Kudosu! by using the thumbs up emote(?) on discussions that were marked as Suggestions or Problems.